incidents_mttr_example.py

```diff
@@ -61,10 +72,13 @@ def main():
         else:
             mttr[field] = {'mttr': d.total_seconds(), 'incidents': 1}
     with open(options.output, 'w') as csvfile:
-        writer = csv.DictWriter(csvfile, fieldnames=['Field', 'Incidents', 'MTTR'])
+        writer = csv.DictWriter(csvfile, fieldnames=[
+                                'Field', 'Incidents', 'MTTR'])
         writer.writeheader()
         for f in mttr:
-            writer.writerow({'Field': f, 'Incidents': mttr[f]['incidents'], 'MTTR': int(mttr[f]['mttr'] / mttr[f]['incidents'] / 60)})
+            writer.writerow({'Field': f, 'Incidents': mttr[f]['incidents'], 'MTTR': int(
+                mttr[f]['mttr'] / mttr[f]['incidents'] / 60)})
```

Comment:
Why break this line at this particular point? Maybe something like this would be more readable?

```python
writer.writerow({
    'Field': f,
    'Incidents': mttr[f]['incidents'],
    'MTTR': int(mttr[f]['mttr'] / mttr[f]['incidents'] / 60)
})
```

Reply:
yes, you are right. fix strange indent (49e8339).